custom-config.fcc.tmpl

```diff
@@ -7,8 +7,6 @@ storage:
       mode: 420
       contents:
         inline: |
-          [Match]
```

Comment:
Remind me, why are we removing this? I remember there was a log around that. I do not remember if the log asked us to add or to remove this

Reply:
Ahh this is wrong, I will put it back.